### ISSUE TITLE
Text

### ISSUE THREAD

**Issue by jsh854** [jsh854/UILab#5]
# Text should be of different sizes 
- h1 - h6
- base font size can be 12px

**Comment by vamsi-spoidey7** [comment]
can you elaborate more. i would like to work on this issue

**Comment by jsh854** [comment]
> can you elaborate more. i would like to work on this issue

sure , this project is going to be ui library , so as every ui library has there own text versions of the simple h1 - h6 items , i want to have the same here but with the ability to have customisations 

e.g h1 should have a default font size but other things like colors, underline, italic (possiblities are endless and depends on how you imagine) 
for base color should be customisable

**Comment by suvraghosh** [comment]
Hey @jsh854 interested to work on this issue if possible can you assign it to me ?

**Comment by jsh854** [comment]
> Hey @jsh854 interested to work on this issue if possible can you assign it to me ?

done

**Comment by vt2730** [comment]
Hi @jsh854 great initiative, would love to work on this. Wanted to know if you are considering documentation (docs) for this from start or later on..(just curious)

**Comment by jsh854** [comment]
> Hi @jsh854 great initiative, would love to work on this. Wanted to know if you are considering documentation (docs) for this from start or later on..(just curious)

yes i am considering that , but initially it would not be ideal - we can think of that once the project takes some pace.
